### PR TITLE
fix: prevents putting meaningless tasks into the queue

### PR DESCRIPTION
**Pull request by boajer**
https://oat-sa.atlassian.net/browse/ADF-270

### CODE REVIEW

File: models/classes/listener/ClassPropertiesChangedListener.php [hunk 1]
```diff
@@ -34,6 +35,9 @@ class ClassPropertiesChangedListener extends ConfigurableService
 
     public function handleEvent(ClassPropertiesChangedEvent $event): void
     {
+        if ( !$this->getServiceLocator()->get(AdvancedSearchChecker::class)->isEnabled()) {
+            return;
+        }
```
Comment (gabrielfs7) [inline models/classes/listener/ClassPropertiesChangedListener.php:40]:
```suggestion
        }
        
```

File: test/unit/model/listener/ClassPropertiesChangedListenerTest.php [hunk 2]
```diff
@@ -84,11 +91,29 @@ public function testRenameClassProperties(): void {
         );
     }
 
+    public function testRenameClassPropertiesNonQueued(): void {
```
Comment (gabrielfs7) [inline test/unit/model/listener/ClassPropertiesChangedListenerTest.php:94]: PSR

```suggestion
    public function testRenameClassPropertiesNonQueued(): void 
    {
```

File: test/unit/model/listener/ClassPropertyRemovedListenerTest.php [hunk 3]
```diff
@@ -38,24 +39,32 @@ class ClassPropertyRemovedListenerTest extends TestCase
     /** @var ClassPropertyRemovedListener */
     private $sut;
 
+    /** @var AdvancedSearchChecker|\PHPUnit\Framework\MockObject\MockObject  */
```
Comment (gabrielfs7) [inline test/unit/model/listener/ClassPropertyRemovedListenerTest.php:42]: FQCN

```suggestion
    /** @var AdvancedSearchChecker|MockObject  */
```

**Review by gabrielfs7 — APPROVED** [review]
LGTM, just some code style fixes

- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Review by chinnu-focaloid — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Pull request merged**